### ISSUE TITLE
FR: Support for OUI28/MA-M and OUI36/MA-S

### ISSUE THREAD

**Issue by ob1kenob** [netdisco/netdisco#1111]
## Expected Behavior

Provide a manufacturer's name when provided a MAC address.
IEEE has begun to assign smaller blocks of MAC addresses, these are referred to as MA-M (Medium, /28) and MA-S (Small, /36).  IEEE are referring to the original "Large" MAC address blocks as MA-L.  Text files of each are available at these URLs:
MA-M: https://standards-oui.ieee.org/oui28/mam.txt
MA-S: https://standards-oui.ieee.org/oui36/oui36.txt

## Current Behavior

Devices show as vendor "IEEE Registration Authority" since that's what's listed in the traditional oui.txt that's been imported.

## Possible Solution

Add two more tables and trigger additional lookups when the vendor = "IEEE Registration Authority" ?
Add further info into existing table and refactor the OUI lookups?
Migrate to the wireshark style of manufacturer table?  https://www.wireshark.org/download/automated/data/manuf

## Steps to Reproduce (for bugs)

1.  Lookup any valid MAC address in the approximately 328 ranges from the OUI.txt that show "IEEE Registration Authority" - one such example of ours is f0:2a:2b:5d:f0:71 - this manufacturer is "Agile Sports Technologies dba Hudl" - they have a MA-M assignment for F0-2A-2B-5

## Your Environment

* Netdisco version used: 2.65.1
* SNMP::Info version used: 3.94

## Config info (deployment.yml)

N/A

## Device information

Nodes, not devices so N/A

**Comment by ollyg** [comment]
thank you @ob1kenob for the info (news to me!) and detailed report. Will look into this soon.

**Comment by ollyg** [comment]
Working on this now ... will go for "Add further info into existing table and refactor the OUI lookups" option.

**Comment by ollyg** [comment]
implemented in 534a9d9 and released in 2.070000
many many thanks @ob1kenob for letting us know about this and explaining so well! ❤️

**Comment by ob1kenob** [comment]
That was quick!  And works well.  Before I had about 300 devices that showed as "IEEE Registration Authority", now I have none!

But I come bearing feedback - I ran the normal "upgrade 2.x" routine that's been on the CPAN install/upgrade webpage.  It appears that cpan didn't deliver the new perl script "ieee-oui-update" into the directory /home/netdisco/perl5/bin/, nor was it linked from /home/netdisco/bin.  Obviously mine is running under user account "netdisco".  I found this all out when netdisco-deploy failed with a pretty specific "I couldn't execute that file" message after the prompt asking to update the OUI's.  Feeling adventurous (and having backups), I simply pulled it from github, saved it into /home/netdisco/perl5/bin, fixed the perms to make it executable (chmod 555), and linked it in the same way all the netdisco-* scripts are.  Re-ran netdisco-deploy and it then completed without errors.  I hope this feedback proves to be helpful!
